python/doc/theory/reliability_sensitivity/form_approximation.rst

```diff
@@ -3,106 +3,75 @@
-     the design point is the point on the limit state boundary the nearest
-     to the origin of the standard space. Thus, :math:`\vect{u}^*` is the
-     result of a constrained optimization problem.
-
-#. | In the standard space, approximate the limit state surface in the
-     standard space by a linear surface at the design point
-     :math:`\vect{u}^*`. Then, the probability :math:`P_f` (:eq:`PfU3`) where
-     the limit state surface has been approximated by a linear surface
-     (hyperplane) can be obtained exactly, thanks to the rotation
-     invariance of the standard distribution :math:`f_{\vect{U}}` :
-
-   .. math::
-     :label: PfFORM
+
+The objective of FORM is to evaluate :math:`P_f`. The method proceeds in three steps:
+
+**Step 1**: Map the probabilistic model in terms of :math:`\vect{X}` thanks to an
+isoprobabilistic transformation :math:`T` which is a diffeomorphism from
+:math:`\supp{\vect{X}}` into :math:`\Rset^n`, such that the distribution of the random vector
+:math:`\vect{U}=T(\vect{X})` has
+the following properties: :math:`\vect{U}` and :math:`\mat{R}\,\vect{U}` have the same distribution for all rotations :math:`\mat{R}\in{\cS\cO}_n(\Rset)`.
+The usual isoprobabilistic transformations are the Generalized
+Nataf transformation and the Rosenblatt one.
+The mapping of the limit state function is
+:math:`G(\vect{U}\,,\,\vect{d}) =  g(T^{-1}(\vect{U})\,,\,\vect{d})`.
+Then, the event probability :math:`P_f` can be written as:
+
+.. math::
+    :label: PfU3
+
+    P_f = \Prob{G(\vect{U}\,,\,\vect{d})\leq 0} = \int_{\Rset^n} \boldsymbol{1}_{G(\vect{u}\,,\,\vect{d}\leq 0}\,f_{\vect{U}}(\vect{u})\,d\vect{u}
+
+
+where :math:`f_{\vect{U}}` is the density function of the distribution in the standard space:
+that distribution is spherical (invariant by rotation by definition). That property implies that :math:`f_{\vect{U}}` is a function of :math:`||\vect{U}||^2` only.
+
+Furthermore, we suppose that outside the sphere which tangents the limit state surface in the standard space, :math:`f_{\vect{U}}` is decreasing.
+
+**Step 2**: Find the design point :math:`P^*` which is the point
+verifying the event of maximum likelihood : the decreasing hypothesis of the standard
+distribution :math:`f_{\vect{U}}` outside the sphere which tangents the limit state surface in
+the standard space implies that the design point is the point on the limit state boundary the
+nearest to the origin of the standard space. Thus, :math:`P^*` is the result of a constrained
+optimization problem.
+
+**Step 3**: In the standard space, approximate the limit state surface in the
+standard space by a linear surface at the design point :math:`P^*`. Then, the probability
+:math:`P_f` :eq:`PfU3` where the limit state surface has been approximated by a linear surface (hyperplane) can be obtained exactly, thanks to the rotation invariance of the standard distribution :math:`f_{\vect{U}}`:
```

Comment:
The next equation uses "E" meaning the CDF of the isoprobabilistic transformation, which is undefined at this point. I would suggest to add a pointer to the "Isoprobabilistic transformations" theory help page (how can we do that?) where the function "E" should be introduced in a general setting.

Reply:
Well done:
![image](https://user-images.githubusercontent.com/31351465/235299573-8ee00156-33b8-4dfe-8ac4-10468a58370e.png)